test/katex-spec.js

```diff
@@ -50,8 +50,13 @@ const _getBuilt = function(expr, settings) {
     // grab the root node of the HTML rendering
     const builtHTML = rootNode.children[1];
 
-    // Remove the outer .katex and .katex-inner layers
-    return builtHTML.children[2].children;
+    // combine the non-strut children of all base spans
```

Comment:
Why?

Reply:
This is for backwards compatibility. Several tests delve into the contents of the children returned from `getBuilt`, for example:

```js
    it("should create ords when at the beginning of lists", function() {
        const built = getBuilt("+ x");

        expect(built[0].classes).toContain("mord");
        expect(built[0].classes).not.toContain("mbin");
    });
```

These tests want to ignore the `.base` elements, but now that there are multiple such elements, their children need to be concatenated together.  The `.strut`s used to be outside the `.base` elements, but now they're inside, so I also strip them away for these tests.

It occurs to me, though, that this simplification could be done in `getBuilt`, not `_getBuilt`.  This would be better.

Reply:
Let's leave things as they are now.  Eventually I'd like to replace those tests with snapshot tests.  We still might want to do some simplification depending on what the snapshots look like.